### PR TITLE
Wrap individual function calls

### PR DESCRIPTION
**Pull request by davedevelopment**
I feel like this gets us slightly closer to something everyone can get a long with with regards to padraic/mockery#464

### CODE REVIEW

**Comment by padraic** [comment]
Any feedback from the Hamcrest side?

**Comment by davedevelopment** [comment]
I have commit access here, I'll merge if @aik099 or @cordoval don't object by tomorrow

File: generator/parts/functions_header.txt [hunk 1]
```diff
@@ -21,3 +21,4 @@ if (!function_exists('assertThat')) {
             $args
         );
     }
+}
```
Comment (cordoval) [inline generator/parts/functions_header.txt]: nice catch!

**Comment by cordoval** [comment]
I don't object at all

**Comment by aik099** [comment]
Travis build passes - I'm happy.

**Comment by davedevelopment** [comment]
Thanks gents!

**Comment by padraic** [comment]
Thanks all!

**Pull request merged**